_inc/lib/class.jetpack-user-event-tracking.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+class Jetpack_User_Event_Tracking {
+
+	private static $_cache = array();
```

Comment:
Drive-by comment since I'm looking at this in concert with tracking on the Calypso side: what's the rationale behind having a local cache for this user meta? The user meta functions already leverage WP's built-in `wp_cache_*` functions, which also persist in memory for a session like this is doing.

Reply:
The rational was that is would be faster. But I see your point and would be happy to remove it.